flink-cdc-common/src/test/java/com/ververica/cdc/common/schema/SelectorsTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.common.schema;
+
+import com.ververica.cdc.common.event.TableId;
+import org.junit.jupiter.api.Test;
+
+import java.util.function.Predicate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/** Test for {@link Selectors}. */
+class SelectorsTest {
+
+    @Test
+    public void testTableSelector() {
+
+        // nameSpace, schemaName, tableName
+        Predicate<TableId> filter =
+                Selectors.tableSelector()
+                        .includeTables("db\\.sc1\\.A[0-9]+,db\\.sc2\\.B[0-1]+")
+                        .build();
+
+        assertAllowed(filter, "db", "sc1", "A1");
+        assertAllowed(filter, "db", "sc1", "A2");
+        assertAllowed(filter, "db", "sc2", "B0");
+        assertAllowed(filter, "db", "sc2", "B1");
+        assertNotAllowed(filter, "db", "sc1", "A");
+        assertNotAllowed(filter, "db", "sc2", "B");
+        assertNotAllowed(filter, "db", "sc1", "AA");
+        assertNotAllowed(filter, "db", "sc2", "B2");
+        assertNotAllowed(filter, "db2", "sc1", "A1");
+        assertNotAllowed(filter, "db2", "sc1", "A2");
+
+        // schemaName, tableName
+        filter = Selectors.tableSelector().includeTables("sc1\\.A[0-9]+,sc2\\.B[0-1]+").build();
```

Comment:
@GOODBOY008 @Jiabao-Sun Could we use `"sc1.A[0-9]+,sc2.B[0-1]+"` here? I mean if `.` is part of regex pattern and then we use `sc1.A\\.*,sc2.B\\.*`.

Reply:
The most common case for entire database sync should be `mydb.users, mydb.orders`, which is better than `mydb\\.users, mydb\\.orders`

Reply:
`sc1.A[0-9]+` can be matched to `sc1.a1` and `sc1.a2`, because `.` can match any character.

Write `mydb.users` and `mydb.orders` directly that can match the corresponding table, but if there is no db constraint, there may be side effects:  `mydb_orders, mydb-orders ...`

Reply:
We can use `\.` if `.` is one part of regex pattern just like we treats `,`, `,` can be part of a regex pattern

Reply:
I mean we can follow same rule and offer a more elegant table id pattern